src/main/java/wolox/training/exceptions/BookIdMismatchException.java

```diff
@@ -0,0 +1,13 @@
+package wolox.training.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import wolox.training.constants.ExceptionConstants;
+
+@ResponseStatus(code = HttpStatus.NOT_ACCEPTABLE, reason = ExceptionConstants.BOOK_ID_MISMATCH)
```

Comment:
In this case I think the BAD_REQUEST is a more clear HTTP status code (The client has to change the request for this to work)
I believe that the NOT_ACCEPTABLE is related to the accepted headers from the request